.gitignore

```diff
@@ -14,3 +14,6 @@
 /vendor/
 
 xdebug.log
+
+
+.idea
```

Comment:
please revert. this should be your local global gitignore config and not this project config. otherwise we will have entries for 20 different IDEs here.